packages/js-dpp/lib/errors/consensus/signature/PublicKeyIsDisabledError.js

```diff
@@ -0,0 +1,24 @@
+const AbstractSignatureError = require('./AbstractSignatureError');
+
+class PublicKeyIsDisabledError extends AbstractSignatureError {
+  /**
+   *
+   * @param {IdentityPublicKey} publicKey
```

Comment:
We can't pass objects like that into consensus error due to serialization. It must be types supported by cbor. I would recommend to pass key ID